DataSetExplorer/Program.cs

```diff
@@ -27,8 +33,91 @@ class Program
         static void Main(string[] args)
         {
             //MakeExcelFromProjectUseCase();
-            FindInstancesRequiringAdditionalAnnotationUseCase();
+            //FindInstancesRequiringAdditionalAnnotationUseCase();
             //FindInstancesWithAllDisagreeingAnnotationsUseCase();
+            ExportAnnotatedDataSet();
+        }
+        
+        private static void ExportAnnotatedDataSet()
```

Comment:
This method would benefit from a communication diagram to understand who is the coordinator and who is responsible for what.